database/layer/admin-persona.js

```diff
@@ -65,3 +61,21 @@ async function updatePersona (tenantName, issClaim, oldSubClaim, newSubClaim, ol
     return `Update failed: ${error.message}`
   }
 }
+
+const insertPersonaByTenantKey = async (tenantKey, subClaim, userContext = '') => {
+  try {
+    const [persona] = await usherDb('personas')
+      .insert({ tenantkey: tenantKey, sub_claim: subClaim, user_context: userContext })
```

Comment:
hey @k1ch shouldn't we check if the tenantKey is a valid tenant key?

Reply:
Yes, invalid tenantkey causes a DB error which we properly handle and return 400 error. Review the test cases.